index.html

```diff
@@ -20,15 +20,15 @@
         <h1 class="name">Hey there. I’m<br>Mariane</h1>
```

Comment:
- [ ] **REQUIRED:** There is a bit much more space between header _(I mean toolbar)_ and section, kindly add space by adding margin or padding to `<h1>` element.

- [ ] **OPTIONAL:** Please make the hamburger button a bit smaller, try to make the same size the portfolio you have chose.